fiftyone/utils/depth_anything.py

```
@@ -0,0 +1,314 @@
+        return {"depth": prediction.depth}
+
+
+class _DepthAnythingV3Transforms:
+    """Input transforms for Depth Anything V3.
+
+    Converts various image input types (file paths, PIL Images, torch Tensors,
+    and numpy arrays) to numpy arrays and tracks original sizes for the output
+    processor.
+    """
+
+    def __call__(self, img):
```

Ruff (0.14.10)

Missing return type annotation for special method `__call__`

(ANN204)